<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shipment</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>521</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOwnerFlavor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listContainer">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>450</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
